docs/buildah-bud.md

```diff
@@ -749,6 +754,12 @@ buildah bud --no-cache --rm=false -t imageName .
 
 buildah bud --dns-search=example.com --dns=223.5.5.5 --dns-option=use-vc .
 
+### Building an multi-architecture image using a --manifest option (Requires emulation software)
+
+buildah bud --arch arm --manifest myimage /tmp/mysrc
+buildah bud --arch x86_64 --manifest myimage /tmp/mysrc
```

Comment:
`x86_64` is not one of the values that's listed in `go tool dist list`'s list of GOOS/GOARCH pairs.  I think it's supposed to be `amd64`.  On a related note, I found https://eloydegen.com/blog/posts/arm-linux-distribtions/ to be tremendously helpful in understanding where we are on 32-bit ARM.

Reply:
Well x86_64 works. I believe. Although this might just ignore bad values.

Reply:
Changed to amd64